src/index.js

```diff
@@ -353,6 +352,52 @@ class OrangeContentScript extends ContentScript {
         '[data-e2e="bh-more-bills"]',
         '[aria-labelledby="bp-historicBillsHistoryTitle"]'
       )
+      await this.runInWorker('processingBill')
```

Comment:
This is a lot code in one function. You should at least have a convertToCozyBill function for example, which would make it more readable